src/main/java/io/druid/data/input/impl/StringInputRowParser.java

```diff
@@ -35,20 +36,35 @@
  */
 public class StringInputRowParser implements ByteBufferInputRowParser
 {
+  private final static Charset defaultCharset = Charsets.UTF_8;
```

Comment:
I implemented it that way initially but made this change based on a previous comment to make it more explicit that UTF_8 is the default charsest